src/nvim/misc1.c

```diff
@@ -2729,38 +2707,33 @@ void fast_breakcheck(void)
   }
 }
 
-/*
- * Get the stdout of an external command.
- * If "ret_len" is NULL replace NUL characters with NL.  When "ret_len" is not
- * NULL store the length there.
- * Returns an allocated string, or NULL for error.
- */
-char_u *
-get_cmd_output (
-    char_u *cmd,
-    char_u *infile,            /* optional input file name */
-    int flags,                 // can be kShellOptSilent
-    size_t *ret_len
-)
+/// Get the stdout of an external command.
+/// If "ret_len" is NULL replace NUL characters with NL. When "ret_len" is not
+/// NULL store the length there.
+///
+/// @param  cmd      command to execute
+/// @param  infile   optional input file name
+/// @param  flags    can be kShellOptSilent
```

Comment:
Essentially a bool, but like this it is maybe more self-documenting. Maybe "can be kShellOptSilent or 0" ?